nengo/learning_rules.py

```diff
@@ -197,6 +201,42 @@ def _argreprs(self):
         return args
 
 
+class Voja(LearningRuleType):
+    """Vector Oja's learning rule.
+
+    Modifies an ensemble's encoders to be selective to its inputs.
+
+    A connection to the learning rule will provide a scalar weight for the
+    learning rate (minus 1). For instance, 0 is normal learning, -1 is no
+    learning, and less than -1 causes anti-learning or "forgetting".
+
+    Parameters
+    ----------
+    learning_rate : float, optional
```

Comment:
Yeah, I just glanced at Connection for reference, and some of its parameters listed in the docstring don't even exist anymore. Should definitely clean this up before release.